source/audioDucking.py

```diff
@@ -179,10 +181,7 @@ def initialize():
 def isAudioDuckingSupported():
 	global _isAudioDuckingSupported
 	if _isAudioDuckingSupported is None:
-		_isAudioDuckingSupported = (config.isInstalledCopy() or config.isAppX) and hasattr(
-			oledll.oleacc,
-			"AccSetRunningUtilityState",
-		)
+		_isAudioDuckingSupported = config.isInstalledCopy() or config.isAppX
```

Comment:
I think we should tear out AppX as we go, it's not risky to remove in parts as it will never be relevant again

```suggestion
		_isAudioDuckingSupported = config.isInstalledCopy()
```